application/templates/mail/request_from_accepted_to_committed.html

```diff
@@ -0,0 +1,8 @@
+<p>A requester on <a href="<%= uiURL %>"><%= appName %></a> isn't sure yet if they
+    want you to fulfill that request.</p>
+<h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
+<p>The requester, <%= requesterNickname %>, isn't ready afterall to have you
```

Comment:
```suggestion
<p>The requester, <%= requesterNickname %>, isn't ready after all to have you
```

Reply:
Oops - who would have thought it's two words. (-;